ckanext/geocat/tests/test_distribution_metadata.py

```diff
@@ -14,7 +14,7 @@
 )
 
 
-class TestGeocatDcatDistributionMetadata(unittest.TestCase):
+class TestGeocatDistributionProtocols(unittest.TestCase):
```

Comment:
The tests so far tested the mapping to the resources in a general way. The tests have been reworked to test the mapping of resources per protocol.